articles/iot-edge/tutorial-deploy-custom-vision.md

```diff
@@ -49,7 +49,7 @@ Before beginning this tutorial, you should have gone through the previous tutori
 
 To develop an IoT Edge module with the Custom Vision service, install the following additional prerequisites on your development machine: 
 
-* [Python](https://www.python.org/downloads/)
+* [Python](https://www.python.org/downloads/) version 3.7.x minimum 
```

Comment:
I am not clear on why this guide requires 3.7.x

Reply:
My bad, my install of Azure IoT SDK for Python was not OK for Python 2.7.
After re installing in the context of 2.7, it is OK, so no need of this comment